From 0b6dcd052cbd3f3a7b214f9208926caa27cf95af Mon Sep 17 00:00:00 2001
From: "Thulasimani,Sivakumar" <sivakumar.thulasimani@intel.com>
Date: Tue, 12 Jan 2016 10:53:02 +0530
Subject: [PATCH 3/5] Revert "FOR_UPSTREAM [VPG]: drm/i915: Update pll values
 for CHT"

This reverts commit 1508298c7938ace412305a54dd8ad3b7a7f6d408.
The values updated as part of this patch were 24bit shifted
where as old ones were 22bit shifted, which required another
patch for the adjustment. This will be replaced with a new
patch that will add the vco values which was the original
purpose of this patch.

Change-Id: Ic05ae41e1db819117799149d3edc78f00fd76702
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-14849
Signed-off-by: Sivakumar Thulasimani <sivakumar.thulasimani@intel.com>
Reviewed-on: https://android.intel.com:443/468192
---
 drivers/gpu/drm/i915/intel_dp.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_dp.c b/drivers/gpu/drm/i915/intel_dp.c
index 56b9711..e72349b 100644
--- a/drivers/gpu/drm/i915/intel_dp.c
+++ b/drivers/gpu/drm/i915/intel_dp.c
@@ -102,11 +102,11 @@ static const struct dp_link_dpll chv_dpll[] = {
 	 * (m2_int << 22) | m2_fraction
 	 */
 	{ DP_LINK_BW_1_62,	/* m2_int = 32, m2_fraction = 1677722 */
-		{ .p1 = 4, .p2 = 2, .n = 1, .m1 = 2, .m2 = 0x2019999A,
-		.vco = 0x76a70 } },
+		{ .p1 = 4, .p2 = 2, .n = 1, .m1 = 2, .m2 = 0x819999a } },
 	{ DP_LINK_BW_2_7,	/* m2_int = 27, m2_fraction = 0 */
-		{ .p1 = 4, .p2 = 1, .n = 1, .m1 = 2, .m2 = 0x1b000000,
-		.vco = 0x83d60 } },
+		{ .p1 = 4, .p2 = 1, .n = 1, .m1 = 2, .m2 = 0x6c00000 } },
+	{ DP_LINK_BW_5_4,	/* m2_int = 27, m2_fraction = 0 */
+		{ .p1 = 2, .p2 = 1, .n = 1, .m1 = 2, .m2 = 0x6c00000 } }
 };
 
 /**
-- 
1.9.1

